docs/development/setup-dapr-development-env.md

```diff
@@ -27,7 +27,7 @@ This document helps you get started developing Dapr. If you find any problems wh
 
 2. Install [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for Go debugging, if desired.
 
-3. Install [golangci-lint](https://golangci-lint.run/usage/install) version 1.55.2.
+3. Install [golangci-lint](https://golangci-lint.run/welcome/install/) version 1.55.2.
```

Comment:
Seems it is not relative to the main topic, please update it in other PR, one thing one PR is good.

Reply:
I'll remove this change, but please don't expect a separate PR from me.

Reply:
Just to be clear, the old link is broken

Reply:
just to be clear, the old link is broken. I've removed this change, but please don't expect a separate PR from me.